devlog/_plan/260902_bug_label_drawdown/071_regaudit_landing.md

```
@@ -0,0 +1,120 @@
+(`OCX_SELF_HOSTED_WINDOWS`) for a trustworthy Windows signal.
+
+## Devlog stack landing
+
+Branch `codex/260902-bug-pr-closeout-stack` (devlog-only) → PR #3218, opened during this
+cycle; merged in the final recount phase so it carries the i3217 record too.
+
+## Bug-label count at the end of this pass
+
+`gh issue list -l bug --state open` = 6, `gh pr list -l bug --state open` = 0. Five are the
+recorded blockers (#3152 needs-repro, #3141 needs-info, #2999 CAS-primitive, #1527 needs-info,
+#1419 needs-info). The sixth, **#3217**, was opened at 2026-09-01T20:39Z while this audit ran:
```

markdownlint-cli2 (0.23.2)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)